app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -81,10 +81,13 @@
                             "ng-model"                => "emsCommonModel.azure_tenant_id",
                             "required"                => "",
                             "checkchange"             => "",
+                            "ng-pattern"              => "/^[A-Fa-f0-9]{8}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{12}$/",
                             "prefix"                  => "default",
                             "reset-validation-status" => "default_auth_status"}
         %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.required"}
           = _("Required")
+        %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.pattern"}
+          = _("Invalid GUID")
```

Comment:
How about:
"Invalid input format, please enter a GUID".
Or
"Invalid input format, please enter a <field_name> in GUID format."